internal/flags.go

```diff
@@ -75,6 +76,23 @@ func objectKindSupportsLabelsFlag(kind manifest.Kind) bool {
 	return ok
 }
 
+var sloSupportingKinds = map[manifest.Kind]struct{}{
+	manifest.KindBudgetAdjustment: {},
+}
+
+func objectKindSupportsProjectSloFlag(kind manifest.Kind) bool {
+	_, ok := sloSupportingKinds[kind]
+	return ok
+}
+
+func registerProjectSloFlag(cmd *cobra.Command, storeSloIn, storeProjectIn *string) {
+	cmd.Flags().StringVarP(storeSloIn, "filters-slo-name", "", "",
```

Comment:
We already have a similar flag in `get alert`, let's keep the pattern :)
 you can extend the flag description to make it more clear, which field is filtered

```suggestion
	cmd.Flags().StringVarP(storeSloIn, "slo", "", "",
```